donate/payments/tasks.py

```diff
@@ -137,8 +137,6 @@ def send_newsletter_subscription_to_basket(data):
 
 
 def send_transaction_to_basket(data):
-    if settings.DONATION_RECEIPT_METHOD == 'DONATE':
-        process_donation_receipt(data)
     send_to_sqs({
```

Comment:
While looking at this portion of code with Pomax, we found that if the "process_donation_receipt" function returns an exception or errors out, the "send_to_sqs" function is never called. To fix this, I moved the SQS function before the process_donation_receipt function, that way in the event that it fails, the information would still get sent to SQS

Reply:
If this can throw, in addition to moving it, let's also make sure to try/except the call so that we can log when it throws.

Reply:
Hi Pomax! Thanks for the heads up, the try/except is being added in https://github.com/mozilla/donate-wagtail/pull/1654. I will request review once I update the sleep to use sched instead 👍